src/plans/details/tabs/Details/components/SettingsSection/components/PlanTransferNetwork/utils/utils.ts

```diff
@@ -0,0 +1,38 @@
+import { ADD, REPLACE } from '@components/ModalForm/utils/constants';
+import { PlanModel, type V1beta1Plan, type V1beta1PlanSpecTransferNetwork } from '@kubev2v/types';
+import { k8sPatch } from '@openshift-console/dynamic-plugin-sdk';
+import { getPlanTransferNetwork } from '@utils/crds/plans/selectors';
+
+import { PROVIDER_DEFAULTS } from './constants';
+
+export const getNetworkName = (value: V1beta1PlanSpecTransferNetwork | null): string => {
+  if (!value || typeof value === 'string') return PROVIDER_DEFAULTS;
+  return `${value?.namespace}/${value?.name}`;
+};
+
+export const transferNetworkToName = (network: V1beta1PlanSpecTransferNetwork | undefined) =>
+  network && `${network.namespace}/${network.name}`;
```

Comment:
Seems like this function is very similar to `getNetworkName`. Can they be combined into 1 or maybe this function could be used inside of `getNetworkName`?

Reply:
used it inside of getNetworkName